test/index.js

```diff
@@ -615,7 +614,7 @@ describe('onBuild()', () => {
     const imageConfigPath = path.join(constants.INTERNAL_FUNCTIONS_SRC, IMAGE_FUNCTION_NAME, 'imageconfig.json')
     const imageConfigJson = await readJson(imageConfigPath)
 
-    expect(imageConfigJson.domains.length).toBe(2)
+    expect(imageConfigJson.domains.length).toBe(1)
```

Comment:
The wikipedia domain was removed, so now there is only one domain. See https://github.com/netlify/next-runtime/pull/1722/files#diff-f3bc73930689da34c69d4dd219fb06b3965a303551a6b4fe527b2492534511e6L74